cypress/apiRequests/Checkout.js

```diff
@@ -1,18 +1,36 @@
-export function createCheckout(
+export function createCheckout({
   channelSlug,
   email,
-  productQuantity,
-  variantsList
-) {
+  productQuantity = 1,
+  variantsList,
+  address,
+  auth = "auth"
+}) {
   const lines = variantsList.map(
     variant => `{quantity:${productQuantity}
                     variantId:"${variant.id}"}`
   );
+  const shippingAddress = address
```

Comment:
We had a utils function for default "" I think?

Reply:
Yes. If I will use it it will cause an error that cannot take fields(city, country, etc.) of undefined. But address is used in few places so I created in utils getDefaultAddress function